.eslintrc

```diff
@@ -4,24 +4,72 @@
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript"
+  ],
+  "plugins": [
+    "prettier",
+    "@typescript-eslint",
+    "import"
   ],
-  "plugins": ["prettier", "@typescript-eslint", "simple-import-sort"],
   "parser": "@typescript-eslint/parser",
   "rules": {
-    "id-length": ["warn", { "min": 2, "exceptions": ["_", "t"] }],
-    "prettier/prettier": ["error", {}, { "usePrettierrc": true }],
-    "simple-import-sort/imports": "error",
```

Comment:
[question] Removing this means that eslint will not be able to auto-fix the imports on save. Is this an intentional change @jmike? That was the core reason why we had this plugin installed. The auto-fix thing.

Reply:
The removal of `simple-import-sort` is intentional because there's an overlap with the `import` plugin. The other changes are wrong - thanks for spotting this.

Autofix should also work with the `import` plugin.